packages/config-array/package.json

```diff
@@ -23,8 +23,9 @@
   },
   "homepage": "https://github.com/eslint/rewrite#readme",
   "scripts": {
-    "build": "rollup -c",
-    "prepare": "npm run build",
```

Comment:
it cannot be removed - we need `prepare/prepublish`  to build before publishing.

Reply:
Oops, good catch!